syntax = "proto3";

package poker;

service AuthService {
    rpc Signin(AuthRequest) returns (AuthResponse);
    rpc Login(AuthRequest) returns (AuthResponse);
}

message AuthRequest {
    string email = 1;
    string password = 2;
    bool cancell = 3;
    string cancellMessage=4;
}

message AuthResponse {
    string responseString = 1;
}
